Apps/WebClient/src/ClientApp/src/views/pcrTest.vue

```diff
@@ -1141,15 +1141,15 @@ export default class PcrTestView extends Vue {
                             Your kit has been registered to your profile.
                         </span>
                         <div class="pt-2">
-                            <router-link to="/">
+                            <router-link to="/logout">
                                 <hg-button
-                                    id="btn-continue"
-                                    aria-label="Continue"
-                                    data-testid="btn-continue"
+                                    id="logoutBtn"
+                                    aria-label="Logout"
+                                    data-testid="logoutBtn"
```

Comment:
Nit: `id` and `data-testid` values should be in kebab case.

Reply:
Agree with mike on the "Log out"

Reply:
Keeping id and data-testid the same as other code.